python/meep.i

```diff
@@ -199,10 +200,26 @@ static int py_list_to_gobj_list(PyObject *po, geometric_object_list *l);
 %feature("immutable") meep::fields_chunk::num_connections;
 
 %include "vec.i"
+%include "meep/mympi.hpp"
 %include "meep.hpp"
 %include "meepgeom.hpp"
 
 extern boolean point_in_objectp(vector3 p, GEOMETRIC_OBJECT o);
 
 %ignore eps_func;
 %ignore inveps_func;
+
+// test for mpi and do requisite setup if present
+%pythoncode %{
+
+if with_mpi():
+    try:
+        from mpi4py import MPI
+    except ImportError:
+        print '\n**\n** failed to load python MPI module (mpi4py)\n**\n'
```

Comment:
The `print` statements will need parentheses to support python 3.  `print('\n**\n* failed ...')`.  Maybe that's why it's failing on travis.